src/llmcompressor/pytorch/model_load/helpers.py

```diff
@@ -51,6 +55,16 @@ def save_checkpoint(
     if processor is not None:
         processor.save_pretrained(save_path)
 
+    # saving the model modifies the model strcuture
+    # as this is only a checkpoint, decompress model to enable future training/oneshot
```

Comment:
I think this makes sense for backward compatibility, we can always revisit later on